packages/flutter/lib/src/material/dropdown.dart

```diff
@@ -1093,6 +1100,12 @@ class DropdownButton<T> extends StatefulWidget {
   /// instead.
   final Color? dropdownColor;
 
+  /// The maximum height of the menu.
+  ///
+  /// The [menuMaxHeight] should be less than or equal to the maximum height of
+  /// one less row, otherwise it is invalid.
```

Comment:
You may also want to specify that if the height is set to a value greater than the allowed max height, then the allowed max height is used by default and explain why that is the case in the documentation as well. This is actually pretty well described [here](https://github.com/flutter/flutter/pull/76493/files#diff-33bd4d1a62f3ed04521278222d46fd03c3bb9c6fed42a022071c6061d064f6c3R323), so just reuse it to some degree